analysis_options.yaml

```diff
@@ -0,0 +1,10 @@
+include: package:lint/analysis_options.yaml
+
+linter:
+  rules:
+    sort_pub_dependencies: false
+
+analyzer:
+  errors:
+    import_of_legacy_library_into_null_safe: ignore
+    file_names: ignore
```

Comment:
maybe just ignore those `Flavorizr` explicit files instead of ignoring the entire project?